subjects/rectperimeter/README.md

```diff
@@ -0,0 +1,32 @@
+# rectperimeter
+
+### Instructions
+
+write a function that accepts two integer arguments, representing the two sides length of a rectangle and returning the perimeter of the rectangle.
+
+your function should return `-1` if one of the arguments is negative.
```

Comment:
remove Code-quotes from 1, 
code-quotes only for : 
- function names
- libraries
- data types
- etc.

Reply:
Done